### ISSUE TITLE
CopyQ window requires two hotkey presses to display

### ISSUE THREAD

**Issue by ayoungethan** [hluk/CopyQ#1939]
Before creating a new issue, see list of [known
issues](https://copyq.readthedocs.io/en/latest/known-issues.html).

**Describe the bug**
The "show main window" shortcut must be pressed twice in order to activate the first window. The first press results in a keyboard shortcut applied to the window currently in focus. The second press successfully activates the main window.

**To Reproduce**
1. Install CopyQ from flatpak on Elementary 6.1
2. Assign a shortcut for activating main window. In my case I use "super+v"
3. Go to another window, such as a text editor
4. Try to activate the main window using super+v. The window will not appear, and only a "v" appears in the text editor. However, if you keep the modifier key held down and press "v" a second time, the window will display correctly.

**Expected behavior**
I expect the main window to open on the first super+v press without entering any keyboard strokes into the currently-focused window. I can get this desired behavior if instead of setting the shortcut via copyq, I set it via the OS keyboard settings shortcut (see below).

**Screenshots**

**Version, OS and Environment**
(Get details from `copyq version` command if possible.)
 - Application Version:
```
CopyQ Clipboard Manager 6.1.0
Qt: 5.15.3
KNotifications: 5.91.0
Compiler: GCC
Arch: x86_64-little_endian-lp64
OS: KDE Flatpak runtime

```
 - Elementary OS 6.1
 - Default environment

**Additional context**
By assigning the shortcut via the OS custom keyboard shortcuts menu in the settings to `flatpak run com.github.hluk.copyq toggle` I do get the desired behavior.

**Comment by hluk** [comment]
> Try to activate the main window using super+v. The window will not appear, and only a "v" appears in the text editor. However, if you keep the modifier key held down and press "v" a second time, the window will display correctly.

This sounds like an issue in the desktop environment (or perhaps "super" keyboard modifier has some extra treatment).

If you are using Wayland (instead of X11), global shortcuts won't work (they may work only from other X11 app depending on the configuration).

**Comment by ayoungethan** [comment]
It is X11, not Wayland yet. The <super> modifier works normally in all other situations, like any other modifier key (alt, shift, ctrl).

Setting the shortcut via the system to execute the command `flatpak run com.github.hluk.copyq toggle` produces the desired result. Using `gtk-launch com.github.hluk.copyq toggle` instead produces different behavior, FYI, launching a window that does not contain the same copy history.

**Comment by hluk** [comment]
I'm not sure if I can do anything about this since it seems that the registered shortcut event is not passed to CopyQ.

Is there any warning or error in the logs (F12 shortcut from the main window or `copyq logs` command)?

**Comment by ayoungethan** [comment]
No, no warning or error thrown. I think you're right, for some reason the first press just isn't being passed to CopyQ.

I suspected this might be the case, but it sounds like just setting the global shortcut via the System Settings is a solid workaround if anyone else has that problem. I thought maybe I was running Wayland but Elementary is still X11 apparently. I filed in part just in case someone else encounters the same issue, there's some documentation of what to do.

Closing this, thank you for helping look into it!
